oak/server/rust/oak_runtime/src/runtime/channel.rs

```diff
@@ -240,11 +240,6 @@ impl Channel {
         self.writer_count.load(SeqCst) > 0
     }
 
-    /// Determine whether there are any users of the channel.
-    pub fn has_users(&self) -> bool {
```

Comment:
Why is this function not necessary anymore?

Reply:
It was already unused, but the compiler couldn't tell that before because the function was accessible outside of the crate.  Now it's internal to the crate, the compiler warns that it's unused.